recipe/cgo/test.sh

```diff
@@ -30,17 +33,18 @@ case $(uname -s) in
     # c.f.: https://github.com/conda-forge/go-feedstock/pull/75#issuecomment-612568766
     pushd $GOROOT; git init; git add --all .; popd
 
+    echo "current architecture is: ${ARCH}"
     case $ARCH in
       ppc64le)
```

Comment:
Regarding ignoring the test results, are the test failures occurring on all architectures?
If no it may be good to set more per architecture conditions.

Also, if relevant, could you add a comment on the kind of failures you see? It could be in the PR description.

Reply:
hmm, sadly not.  it is a raise condition about a test and its timout (which is hard-coded).  So sometimes it succeeds, sometimes it get killed)